pkgs/development/python-modules/typedload/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  attrs,
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pytestCheckHook,
+  python,
+  setuptools,
+}:
+
+let
+  typedload = buildPythonPackage rec {
+    pname = "typedload";
+    version = "2.37";
+    format = "pyproject";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "DNYPufshGeAGQ7wwLs5nANoxRtFFpsRaOeLsgVHXQLE=";
+    };
+
+    meta = with lib; {
+      homepage = "https://ltworf.github.io/typedload/";
+      description = "Load and dump json-like data into typed data structures";
+      license = licenses.gpl3;
+      maintainers = with maintainers; [ ppentchev ];
+    };
+
+    propagatedBuildInputs = [ setuptools ];
```

Comment:
```suggestion
    build-system = [ setuptools ];
```

Reply:
Fixed in 0f9707085d655e1fc51539fb1f0264e5ac48767d; thanks!